UI.py

```diff
@@ -237,18 +237,32 @@ def CreateGetLeavesAcrossRangeOptions():
 
     return member_options
 
-def CreateMultipleLeavesEmbed(leaves_group, include_reason):
-    embed = discord.Embed(
-        title = f'Applied Leaves',
-        description = f'<@!{leaves_group[0][0]["member_id"]}> has requested:',
-        colour = 0x4682B4
-    )
-    embed.set_thumbnail(url = os.getenv("Leave_Balance_Link"))
-    embed.add_field(name = '\u200B', value = '\u200B', inline = False)
+def CreateMultipleLeavesEmbed(leaves, include_reason):
+    embeds_to_send = []
```

Comment:
`to_send` doesn't make sense here. You're assuming the caller, don't do that. Simply name this `output_embeds` or even `embeds` or `output`.

Reply:
Done

Reply:
You've completely missed the point here Abdo, in the context of this function, they don't make sense being named as `to_send`, this function simply creates the embeds, it shouldn't care about whether something is to be sent or not. 

Moreover, in the main function where this is called, it makes sense to call it `to_send` as in that context we are sending these said embeds. 

If a comment doesn't make sense to you, please clarify it so we don't wastefully go back and forth like this

Reply:
no you are right, i rename the var in the main script and did not rename it here, totally missed it mb